tool/tsh/common/proxy_test.go

```diff
@@ -249,6 +251,60 @@ func TestSSHLoadAllCAs(t *testing.T) {
 	}
 }
 
+// TestWithRsync tests that Teleport works with rsync.
+func TestWithRsync(t *testing.T) {
+	_, err := exec.LookPath("rsync")
+	require.NoError(t, err)
+
+	s := newTestSuite(t)
+	tshHome, _ := mustLogin(t, s)
+
+	// create ssh client config rsync will use
+	ctx := context.Background()
+	var buf bytes.Buffer
+	err = Run(ctx, []string{"config"}, setHomePath(tshHome), setOverrideStdout(&buf))
+	require.NoError(t, err)
+
+	dir := t.TempDir()
+	configPath := filepath.Join(dir, "ssh_config")
+	err = os.WriteFile(configPath, buf.Bytes(), 0o777)
```

Comment:
nit: It's probably closer to the reality
```suggestion
	err = os.WriteFile(configPath, buf.Bytes(), 0o600)
```

Reply:
Done